src/components/dialogs/cancel.vue

```diff
@@ -64,11 +65,11 @@ export default class CancelDialog extends Mixins(NameRequestMixin, PaymentMixin)
   }
 
   private get isVisible (): boolean {
-    return PaymentModule[PaymentTypes.CANCEL_MODAL_IS_VISIBLE]
+    return this.$store.getters['cancelModalIsVisible']
```

Comment:
So this is what you were talking about.

It feels too "low level" or detailed, but if it helps for now, ok.

Can you (re)use a type for the property name?

Reply:
It does for sure. 

I bet we can, good idea!